src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -420,8 +420,13 @@
         color: $clr-table-font-color;
         text-align: left;
         flex: 1 1 auto;
-        align-self: flex-start;
+        align-self: center;
```

Comment:
I believe you also need `align-items: baseline;` here. 

Try the demo app with something like this to see what I was looking at when I tested the changes and they didn't work for the colun with a signpost in it:

```
<clr-signpost>
  <clr-signpost-content *clrIfOpen>
    <h3>Classless Inter-Domain Routing (CIDR)</h3>
    <p>The routing prefix may be expressed in Classless Inter-Domain Routing (CIDR) notation written as the first address of a network, followed by a slash character (/), and ending with the bit-length of the prefix. For example, 192.168.1.0/24
      is the prefix of the Internet Protocol Version 4 network starting at the given address, having 24 bits allocated for the network prefix, and the remaining 8 bits reserved for host addressing. The IPv6 address specification 2001:db8::/32
      is a large address block with 296 addresses, having a 32-bit routing prefix.</p>
  </clr-signpost-content>
</clr-signpost>
```

Reply:
I actually tested this with https://stackblitz.com/edit/datagrid-header-signpost-spacing-issue-1279-leika5, it looks good though.

Adding `align-items: baseline;` makes no difference.

Reply:
It looks like the stackblitz is running Clarity v0.12 but the fix is for Clarity v2.x.x. Is this fix for v0.12? 

There have been a lot of changes to the css and datagrid templates since v.0.12.x. If this fix is for master it should be checked against the datagrids used for the dev app on the master branch since they reflect the current state of css, templates and DOM structure. Using the dev app with the fix as it is currently coded I see this:

<img width="1161" alt="Screen Shot 2019-09-03 at 1 46 22 PM" src="https://user-images.githubusercontent.com/433692/64207585-c425df80-ce51-11e9-97bd-85addb11e269.png">


Adding `align-items: baseline;` to the `datagrid-column-title` class produces this:

<img width="1153" alt="Screen Shot 2019-09-03 at 1 47 00 PM" src="https://user-images.githubusercontent.com/433692/64207599-cbe58400-ce51-11e9-81fb-468b2bf86e56.png">

The column text for the signpost column should be vertically centered similar to the column text in the other columns that do not have a signpost in them.

Reply:
Ahh, I got it. The fix was meant for master, I will check the change against the dev app.

Reply:
added `align-items: baseline` and verified in the dev app.